x-pack/plugins/infra/public/components/asset_details/tabs/overview/metadata_summary/metadata_summary_list.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiDescriptionList,
+  EuiDescriptionListDescription,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+import type { InfraMetadata } from '../../../../../../common/http_api';
+import { NOT_AVAILABLE_LABEL } from '../../../translations';
+import { useTabSwitcherContext } from '../../../hooks/use_tab_switcher';
+import { FlyoutTabIds } from '../../../types';
+import { ExpandableContent } from '../../../components/expandable_content';
+import { MetadataHeader } from './metadata_header';
+
+interface MetadataSummaryProps {
+  metadata: InfraMetadata | null;
+  metadataLoading: boolean;
+}
+
+export interface MetadataData {
+  field: string;
+  value: string | string[] | undefined;
```

Comment:
nit:

`value: string | string[] | undefined;`
```suggestion
  value?: string | string[]
```